test/models/test_calculator.jl

```diff
@@ -26,18 +26,46 @@ elements = (:Si, :O)
 level = M.TotalDegree()
 max_level = 15
 order = 3 
+E0s = (-158.54496821, -2042.0330099956639)
+NZ = length(elements)
+
+function make_rin0cut(zi, zj) 
+   r0 = ACE1x.get_r0(zi, zj)
+   return (rin = 0.0, r0 = r0, rcut = 6.5)
+end
+
+rin0cuts = SMatrix{NZ, NZ}([make_rin0cut(zi, zj) for zi in elements, zj in elements])
+
 
 model = M.ace_model(; elements = elements, order = order, Ytype = :solid, 
                       level = level, max_level = max_level, maxl = 8, 
                       pair_maxn = 15, 
-                      init_WB = :glorot_normal, init_Wpair = :glorot_normal)
+                      init_WB = :glorot_normal, init_Wpair = :glorot_normal,
+                      E0s = E0s,
+                      rin0cuts = rin0cuts
+                      )
 
 ps, st = LuxCore.setup(rng, model)
 
 calc = M.ACEPotential(model, ps, st)
 
 ##
 
+@info("Testing correctness of E0s")
+ps_vec, _restruct = destructure(ps)
+ps_zero = _restruct(zero(ps_vec))
+
+for ntest = 1:20
+   local Rs, Zs, z0, at, efv 
+   at = AB.rattle!(AB.bulk(:Si, cubic=true) * 2, 0.1)
```

Comment:
we should improve this test by also "rattling" the elements. I'll try to make a  PR to AtomsBuilder today and we can then update this test before merging.

Reply:
P.S.: this is now running through the general registry. I'll ping you again when it is ready.

Reply:
Now registered in AtomsBuilder 0.0.4

Reply:
I adjusted this test.